tests/wake-format/basic/pass.sh

```diff
@@ -0,0 +1,3 @@
+#! /bin/sh
+
+../../../bin/wake-format.native-cpp14-release basic.wake
```

Comment:
Anyone have any ideas on how to do this without making major changes to `tests/tests.wake`?

Reply:
You can add a `$2` to the testing framework to be wake-fromat. That's the only real hack here. wake-unit is even more special because `wake-unit` doesn't wind up in the install like we want wake-format to.

Really we need to redesign the output API of wake's build I think. It would be better if `$1` was an install directory so you'd say `$1/bin/wake` and then we had `github.com/wake:/native-cpp14.release/bin/wake` as the default wake location so that each variant gave you a release and then `$1` could be gathered from the variant in the test code. That's a bit of major yak shave thoguh.